datasets/us/me/med_exclusions/us_me_med_exclusions.yaml

```
@@ -0,0 +1,40 @@
+title: US Maine Medicaid Excluded Provider List
+entry_point: crawler.py
+prefix: us-medme
+coverage:
+  frequency: daily
+  start: "2024-11-27"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  This dataset contains a list of providers who have been excluded from Maine's Medicaid program.
+description: |
+  MaineCare Services publishes a monthly report listing providers excluded from the state Medicaid
+  program. Since the report is updated monthly, it may not include providers excluded within the
+  last 30 days. In such cases, MaineCare Services recommends conducting a name search using their
+  [portal](https://mainecare.maine.gov/mhpviewer.aspx?FID=MEEX).
+publisher:
+  name: Maine Department of Health and Human Services
+  description: |
+   > The Maine Department of Health and Human Services (DHHS) is dedicated to promoting health,
+   > safety, resilience, and opportunity for Maine people. The Department provides health and
+   > social services to approximately a third of the State's population, including children,
+   > families, older Mainers, and individuals with disabilities, mental illness, and substance
+   > use disorders.
```

`|` (literal block scalar) is sufficient to preserve line breaks exactly as they are in the input, and it does not require `> ` to work
```suggestion
    The Maine Department of Health and Human Services (DHHS) is dedicated to promoting health,
    safety, resilience, and opportunity for Maine people. The Department provides health and
    social services to approximately a third of the State's population, including children,
    families, older Mainers, and individuals with disabilities, mental illness, and substance
    use disorders.
```

---

Also for a better rendering of the website, it is best to keep the length of each line under 90 characters